### ISSUE TITLE
how to set up  Graph model to monitor all task-wise loss?

### ISSUE THREAD

**Issue by rex-yue-wu** [fchollet/keras#580]
Hi,

When I train a multitask CNN, I am more interested in monitoring the overall loss and also each individual loss of a task. For example, instead of seeing
```
9860/9860 [=============================>] - 13s - loss: 0.0064 - acc: 0.0971 - val_loss: 0.0088 - val_acc: 0.0567
```
I want to see
```
9860/9860 [=============================>] - 13s - loss: 0.0064 - acc: 0.0971 -task1loss: 0.xxxx -task2loss: 0.yyyy - val_loss: 0.0088 - val_acc: 0.0567 -val_task1loss: 0.aaaa -val_task2loss: 0.bbbb
```
Does anyone know how to set up this?

**Comment by fchollet** [comment]
You should train your model batch by batch (or epoch by epoch if you are fine with just having one data points per epoch), and every time you need to check the loss of your individual tasks, do something like:

```python
preds = model.predict(X)
losses = {}
for k in preds:
   losses[k] = np.abs(y[k] - preds[k]) # this is MAE for instance
# then print it
# you'll find utils for numpy-side loss computation in utils/np_utils.py
```

**Comment by MJahanageerQureshi** [comment]
is there any way we could use the above example using keras callbacks
